lib/core/src/lib/datatable/components/datatable/datatable.component.spec.ts

```diff
@@ -1844,6 +1844,15 @@ describe('Column Resizing', () => {
         expect(adapter.setColumns).toHaveBeenCalledWith(columns);
     }));
 
+    it('should prevent default behaviour on row click event', () => {
+        const clickEvent = new MouseEvent('click');
+        spyOn(clickEvent, 'preventDefault');
+        const resizeHandle: HTMLElement = getResizeHandler();
```

Comment:
no need to set type because type is set based on return type of getResizeHandler function

Reply:
removed